src/webview/components/options-widget.tsx

```diff
@@ -238,6 +234,10 @@ export class OptionsWidget extends React.Component<OptionsWidgetProps, {}> {
                                 onChange={this.handleAdvancedOptionsDropdownChange}
                                 options={allowedGroupsPerRow}
                                 className="advanced-options-dropdown" />
+                            <Button
```

Comment:
Is it easy to identify if the current configuration is the same as the global configuration? If yes, then I would recommend to show this button only if the configurations do not match.

Reply:
Good idea! However, it is not super easy without changing more than a few lines, as we don't distinguish between the first values of properties, the ones that are then set by webview main once read from preferences, and then the ones changed by the user.
The only simple approach would be to keep a boolean flag that we switch on first change and only the enable the button after the first change. Do you think it is worth doing that?

Reply:
The button is tempting me to click it :) Maybe instead of hiding it we can create an icon button (e.g., [refresh](https://microsoft.github.io/vscode-codicons/dist/codicon.html)) at the top right corner of the overlay panel?

Reply:
Yeah, I see, then it would be less prominent and not so tempting to consider it as a confirm button.
I think a button with `codicon discard` in the top right corner may be the best option.
I'll try to implement that later today or early next week.